src/http/ContentLengthInterpreter.h

```diff
@@ -19,12 +21,21 @@ namespace Http
 class ContentLengthInterpreter
 {
 public:
-    explicit ContentLengthInterpreter(const int aDebugLevel);
+    ContentLengthInterpreter();
 
     /// updates history based on the given message-header field
     /// \return true iff the field should be added/remembered for future use
     bool checkField(const String &field);
 
+    /// prohibits Content-Length in 1xx and 204 responses
+    void applyStatusCodeRules(const StatusCode code) { prohibitedAndIgnored = ProhibitsContentLength(code); }
```

Comment:
I think this method should not make prohibitedAndIgnored false. This and similar methods can make prohibitedAndIgnored true, but they should not undo the true setting by another, earlier method.

Reply:
Done.